Makefile

```diff
@@ -6,22 +6,42 @@ API_GROUPS := metacontroller/v1alpha1
 PACKAGE_VERSION ?= $(shell git describe --always --tags)
 OS = $(shell uname)
 
+PKGS = $(shell go list ./... | grep -v '/test/integration/\|/examples/')
 ALL_SRC = $(shell find . -name "*.go" | grep -v -e "vendor")
 
 BUILD_LDFLAGS = -X $(PACKAGE_NAME)/build.Hash=$(PACKAGE_VERSION)
 GO_FLAGS = -gcflags '-N -l' -ldflags "$(BUILD_LDFLAGS)"
+GOPATH := $(firstword $(subst :, ,$(GOPATH)))
+PATH := "$(PWD)/hack/bin:$(PATH)"
 
 REGISTRY ?= quay.io/amitkumardas
 IMG_NAME ?= metac
 
+# download tools if necessary
+ifeq (, $(shell which controller-gen))
```

Comment:
The build is failing somewhere around here. hack/update-codegen.sh needs to be run from inside the vendor directory to generate the clientset, informers, etc. 

@luisdavim can you run `make generated_files` on your setup & confirm if there are any errors?
In addition, what is the original error you are getting on your setup without these changes?

Below is my setup details:
```sh
  metac > pwd
/home/amit/rustdev/go/openebs.io/metac
  metac > echo $GOPATH
/home/amit/work
  metac > go version
go version go1.12 linux/amd64
```

Reply:
yes, it seems to work on my environment but fails when running from docker, it complains it can't find `bash`...

Reply:
the problem was actually some extra quotes in the `PATH`, a copy paste issue when moving it from a command in the make targets to a variable in the `Makefile`.